src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/ApplicationPropertyPage.xaml

```diff
@@ -176,6 +179,24 @@
     </DynamicEnumProperty.Metadata>
   </DynamicEnumProperty>
 
+  <!-- TODO: Shortlink for help URL -->
+  <BoolProperty Name="UseApplicationFramework"
```

Comment:
I know wording it's confusing, but this property actually translates into `MyType` in the project file.

Reply:
I see that now. Thanks! Looks like:
- `true` = `<MyType>WindowsForms</MyType>`
- `false` = `<MyType>WindowsFormsWithCustomSubMain</MyType>`

Reply:
Right! We probably will need to intercept that boolean value to convert it into the actual string.

Reply:
Looking into it, it seems that `MyType` is more involved. It feels similar to `Output Type`, but specifically relates to the VB `My` concept. It feels like we should actually have `MyType` as its own property entirely. It being changed related to this concept of 'application framework' is just an artifact of that concept, not the `MyType` property itself. https://docs.microsoft.com/en-us/dotnet/visual-basic/developing-apps/customizing-extending-my/customizing-which-objects-are-available-in-my#predefined-_mytype-values